rollup.test.config.js

```diff
@@ -0,0 +1,9 @@
+import babel from 'rollup-plugin-babel';
+import multiEntry, { entry } from 'rollup-plugin-multi-entry';
+
+export default {
+  entry,
+  plugins: [babel(), multiEntry('test/**/*_test.js')],
```

Comment:
We only have one test file in this project, but this allows adding more and having them all included automatically in the test bundle. I don't really like that we end up with an intermediate build product for tests, but it's the only way to easily test things not exposed directly by the bundle.